testing/run-scripts/serve-zork-metrics.sh

```diff
@@ -0,0 +1,4 @@
+docker build -t zork-metrics ../../monitor
```

Comment:
renamed! I was getting auth errors when trying to push to Docker Hub. Do I need my docker account (lucyhe) added to the uproxy organization (https://hub.docker.com/u/uproxy/) so I can publish?